_pages/en/product-fw-guidance.md

```diff
@@ -0,0 +1,352 @@
+This is the new opportunity that Digital brings: the ability to quickly act on those insights, using evidence-based decision making.
+All with the objective to improve the results that Programs are being evaluated against via the [TB Policy on Results](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=31300).
+For example, Job Bank's results being displayed [in GC's InfoBase](https://www.tbs-sct.gc.ca/ems-sgd/edb-bdd/index-eng.html#infographic/program/HRSD-BGO05/results).
+
+In the traditional Program-Corporate relationship, IT gets funding to “keep the lights on” and projects are used when Programs want to engage IT staff to make software changes. This can create unnecessary lag for much needed small improvements.
+
+<figure>
+<a href="{{ site.baseurl }}/assets/images/seb-project-wall-iitb.png">
+        <img class="center" src="{{ site.baseurl }}/assets/images/seb-project-wall-iitb.png"
+        alt="The image contains two org charts separated by a wall. The org chart on the left of the wall has the label SEB while the one on the right has the label IITB. Over the wall, an arrow goes from SEB org chart to IITB org chart with the text Projects = Change."
+        />
+  </a>
+  <figcaption>Figure 2. Projects are used to orchestrate change</figcaption>
+</figure>
+
+In the digital world a departmental product “team”, like Job Bank, should embed both business and IT staff.
+However, because Treasury Board Policies regulating Programs and Corporate put different persons in charge of IT and Program functions, these two separate persons will naturally want the staff responsible for their function in their respective org charts.
+
+This is where we want to differentiate improvements from projects and enable funding teams with cross-branch memberships.
+
+<figure>
+<a href="{{ site.baseurl }}/assets/images/continuous-improvements-vs-projects.png">
+        <img class="center" src="{{ site.baseurl }}/assets/images/continuous-improvements-vs-projects.png"
+        alt="The image contains three rectangles, each with a different saying. All three refer to a different kind of funding coverage. The one on the left has an image of a person running with the label Operate (run) maintain value. It is about operation and maintenance and refers to A-Base funding. The second rectangle on its right has an image of a spinning arrow going upwards with the label Enhance (Improve) Drive Value. The text under that image says Continuous Improvements, small growth and also refers to A-Base funding. The third rectangle on the right has an image of a light bulb with the label Innovate (Change) Create Value. The text in it says Innovate, Large Growth, Transform, Procurement and it refers B-Base funding."
+        />
+  </a>
+  <figcaption>Figure 3. Three types of change and their funding coverage</figcaption>
+</figure>
+
+Funding RUN activities continue to be used for “keeping the lights on”.

```

Comment:
Wondering if this should be 'Funding for RUN activities...'. If yes, 'continue' should be 'continues'. As in 'Funding for RUN activities continues to be used for...'.